applications/serial_lte_modem/doc/CARRIER_AT_commands.rst

```diff
@@ -107,6 +108,16 @@ The response syntax depends on the commands used.
 Examples
 ~~~~~~~~
 
+::
+
+   AT#XCARRIER="auto_connect"
```

Comment:
Why auto_connect does not have "read" and "write" actions when all other writable parameters have those?

Reply:
No special reason.
 AT#XCARRIER="auto_connect"  ......... READ
 AT#XCARRIER="auto_connect",<value>  ..... WRITE then READ

Reply:
I suggest that the format is constant over all parameters